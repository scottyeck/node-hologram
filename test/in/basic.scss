/*doc
---
title: Basic
name: 01-basic
category: testing
---

Hello, world!
*/