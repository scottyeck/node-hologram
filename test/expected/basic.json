[{
	"name": "testing",
	"blocks": [{
		"title": "Basic",
		"id": "01-basic",
		"content": "Hello, world!",
		"category": "testing"
	}]
}]